### PR TITLE
chore: create openapi.json with openapi 3.1.0 format

### PR DESCRIPTION
**Pull request by mertssmnoglu**
Create `openapi.json` under the Server directory. Yaml is easy to read and write for me but to make it compatible with **SwaggerHub** and **SwaggerUIExpress** I choose the json.

```js
const openapi = require("./openapi.json")
```

It's not a complete fix but still associated with #581 

## To Do List
- [x] info
  - [x] title, summary, description
  - [x] contact
  - [x] version
- [x] servers
  - [x] local development server
- [x] tags(route groups) 
- [ ] paths
  - [x] /auth/register
  - [x] /auth/login
  - [ ] ...
  - [x] /healthy

### CODE REVIEW

**Comment by mertssmnoglu** [comment]
I filled contact email with `hello@bluewavelabs.ca` but is there a specific support email for it?

**Comment by gorkem-bwl** [comment]
> I filled contact email with `hello@bluewavelabs.ca` but is there a specific support email for it?

You can use support@bluewavelabs.ca for this, eventually both are catch-all mailboxes :-)

File: Server/openapi.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+  "openapi": "3.1.0",
+  "info": {
+    "title": "BluWave Uptime",
```
Comment (ajhollid) [inline Server/openapi.json:4]: Should be `Bluewave Uptime"

**Review by ajhollid — COMMENTED** [review]
Great start :+1:

**Comment by mertssmnoglu** [comment]
I don't want to make this PR bigger than that. I will continue to contributing with smaller and targeted PRs.

**PR Summary**:

Users wishing to use Swagger in their local machines can run a Swagger Web UI with this `openapi.json`. They can create a user, login with their credentials and check the API health.

```js
const express = require("express")
const swaggerUIExpress = require("swagger-ui-express")
const app = express()
const json = require("./openapi.json")

app.use("/openapi", swaggerUIExpress.serve, swaggerUIExpress.setup(json))

app.listen(process.env.PORT, () => {
  console.log("Listening the server on ", process.env.PORT)
})
```

**Review by ajhollid — APPROVED** [review]
Thank you for your contribution here, this is a great start for us to build on.

**Pull request merged**